Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -195,8 +195,13 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
 
             recreateSectionsWithCurrentItems()
 
-            let vc = ViewControllerFactory.importOwnerViewController(presenter: self)
-            present(vc, animated: true)
+            var controller: UIViewController
+            if App.configuration.toggles.walletConnectOwnerKeyEnabled {
+                controller = ViewControllerFactory.selectKeyTypeViewController(presenter: self)
+            } else {
+                controller = ViewControllerFactory.importOwnerViewController(presenter: self)
+            }
```

Comment:
the select key type view controller should move to the "add or create key" controller

Reply:
done